compiler/rustc_span/src/symbol.rs

```diff
@@ -2000,6 +2002,8 @@ symbols! {
         unsafe_no_drop_flag,
         unsafe_pin_internals,
         unsize,
+        unsized_const_param_ty,
+        unsized_const_parameters,
```

Comment:
```suggestion
        unsized_const_params,
```

what's up w the inconsistent naming?

Reply:
no real reason, just figured it was less important for a feature gate to be short than a trait/derive that you'd write a bunch

Reply:
it's less that it needs to be short or long, it's just that it's pretty inconsistent with the other feature gate that users are likely familiar w/ if they reach for this one. i'd prefer if we change it

Reply:
yeah it makes sense to keep them the same as eachother